packages/core/core/src/types.ts

```diff
@@ -264,8 +264,8 @@ export interface ProductGetters<PRODUCT, PRODUCT_FILTER> {
   getCategoryIds: (product: PRODUCT) => string[];
   getId: (product: PRODUCT) => string;
   getFormattedPrice: (price: number) => string;
-  getTotalReviews: (product: PRODUCT) => number;
-  getAverageRating: (product: PRODUCT) => number;
+  getTotalReviews?: (product: PRODUCT) => number;
```

Comment:
they can't be optional, if it's not supported by shopify now, please add them with a logger that mentions about missing implementation and it will be provided in the future - always bottom-up: core affects the integration, not integration core